sdk/storage/Azure.Storage.DataMovement.Blobs/README.md

```diff
@@ -81,17 +64,66 @@ We guarantee that all client instance methods are thread-safe and independent of
 
 ## Examples
 
-Please see the examples for [Blobs][blobs_examples].
+### Examples using BlobContainerClient extension methods to upload and download directories.
+
+Instantiate the BlobContainerClient
+```C# Snippet:ExtensionMethodCreateContainerClient
+BlobServiceClient service = new BlobServiceClient(serviceUri, credential);
+
+BlobContainerClient container = service.GetBlobContainerClient(containerName);
+```
+Upload local directory to the root of the container
+```C# Snippet:ExtensionMethodSimpleUploadToRoot
+DataTransfer transfer = await container.StartUploadDirectoryAsync(localPath);
 
-### Start Upload from Local File to Block Blob
+await transfer.AwaitCompletion();
+```
+Upload local directory to a directory in the container by specifying a directory prefix
+```C# Snippet:ExtensionMethodSimpleUploadToDirectoryPrefix
+DataTransfer transfer2 = await container.StartUploadDirectoryAsync(localPath, blobDirectoryPrefix: Path.GetDirectoryName(localPath));
```

Comment:
can we get our sample files that are populating this reorganized a bit so we don't have some samples in the readme using this `transfer2` name? Kinda odd from the reader's perspective.